### ISSUE TITLE
Add "chronic" gem to deal with ambiguous dates?

### ISSUE THREAD

**Issue by dannguyen** [TheUpshot/statement#87]
I know [mojombo/chronic] is a big gem, but it smoothes a few things with date parsing. In particular, it has some conveniences for ambiguous dates.

Example: [Sen. Meehan's Drupal-views-powered press release page which outputs dates in the ambiguous format of __"03 Feb"__](https://meehan.house.gov/media-center/press-releases?page=0)  (issue: https://github.com/TheUpshot/statement/issues/84, my recent commit: https://github.com/dannguyen/statement/commit/91d188e9ff689b3db1b1054802c81f4c1fa903ab)

A possible solution is that, for each page of press releases, download and parse the __first__ result, extract the full date, then use chronic's [`:context` or `:now`](https://github.com/mojombo/chronic#usage) arguments to disambiguate the dates for the rest of the list of press releases. It's possible to hand-roll a solution, but chronic is probably more stable without being too much of a pain of a dependency.

**Comment by dwillis** [comment]
Yeah, this is a problem with some other sites as well. So far I've loaded each release's URL to get the full date, which is on that page, but this is something worth discussing.

**Comment by dwillis** [comment]
An update: Meehan's pages can be scraped with the date by using our Drupal scraper.
